google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/BucketIamSnippets.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * EDITING INSTRUCTIONS
+ * This file is referenced in Storage's javadoc. Any change to this file should be reflected in
+ * Storage's javadoc.
```

Comment:
The only place I found in Storage javadoc which references the snippet java file is here:https://github.com/GoogleCloudPlatform/google-cloud-java/blob/f6b8dda6d20d3e029051c1136d521586b33cacaf/google-cloud-storage/src/main/java/com/google/cloud/storage/package-info.java#L33

I think you can remove this instruction comment here.